DecodePerfToJson/PerfToJson.cs

```diff
@@ -166,7 +166,7 @@ private void WriteFromReader()
                         // Non-EventHeader decoding.
                         if (this.InfoOptions.HasFlag(PerfInfoOptions.N))
                         {
-                            this.JsonWriter.WriteString("n", sampleEventInfo.Name);
+                            this.JsonWriter.WriteString("n", sampleEventInfo.GetName()); // Garbage
```

Comment:
Garbage?

Reply:
This line of code can potentially allocate a string, which is used and then thrown away. Not a big deal, but I try to minimize that during data processing. I add a comment to lines that allocate garbage so I can keep track of them. In this case I don't think it's worth avoiding the allocation, just worth noting.

And yes, the code path for decoding a perf.data file is mostly garbage-free. Allocations per-file, but essentially no allocations per-event.